### ISSUE TITLE
[suggestion] Add an option "Always translate using [service name]" for it to work globally

### ISSUE THREAD

**Issue by rado84-github** [FilipePS/Traduzir-paginas-web#514]
I have a suggestion: add an option in the extension settings to always use one of the two services used for translation. I visit many websites whose language isn't English, I can only imagine I'm not the only one who visits such sites and it can become annoying if I have to manually choose a translation service for each website. It would be a lot easier if we can choose the translation service globally. I took a screenshot of the settings and added the yellow text as an example of what I'm suggesting:

![Picture_created_16-10-2022_10-31-25](https://user-images.githubusercontent.com/41172201/196023964-72be3748-d540-4a65-b808-5fd74fa8c37e.png)

**Comment by FilipePS** [comment]
When you change the translation service, the change is applied to all sites and languages.

What you would like is to set the service individually for each site, so that the configuration is saved so when you open the site again, use the service selected earlier?

Or specify the translation service to use according to the site's original language. For example: use Yandex for Russian sites and Google for German sites?

**Comment by rado84-github** [comment]
> When you change the translation service, the change is applied to all sites and languages.
> 
> What you would like is to set the service individually for each site, so that the configuration is saved so when you open the site again, use the service selected earlier?
> 
> Or specify the translation service to use according to the site's original language. For example: use Yandex for Russian sites and Google for German sites?

No, global service for all sites will be enough. The problem is I can't see an option to select the service in settings:

![Picture_created_19-10-2022_18-40-30](https://user-images.githubusercontent.com/41172201/196739209-4865f00f-395f-408c-92b0-1552a64588a2.png)

Google's translation usually is a joke, whereas Yandex's is often better. And lately Russians must have decided to joke with Google translation bc I noticed that someone has set the Russian word for "seeders" (usually seen on torrent trackers) to be translated as "Urine", which I'm guessing you know what it means... That's why I wanna try changing the translation service globally to Yandex for all sites. But as I said - there's no such an option in settings.

**Comment by rado84-github** [comment]
I found where the button to change the service was. Not where I expected it to be, though.

**Comment by Coffee-Viking** [comment]
@FilipePS I actually came to ask for a related feature, with the twist that I would like to set a fixed translation service for a set language. i.e. I would like to use google overall, but Yandex for Chinese. Reason being is that google services are blocked in China, while Yandex is not. On the other hand side, many Chinese websites are not accessible from abroad, so I have to use them without a VPN, rendering google's translation useless. It's a minor issue since switching the service can be done with the click of a button, but it would be nice to have the option to assign a fixed language-translator-pair.
